src/cpp/client/create_channel_posix.cc

```diff
@@ -34,8 +34,6 @@ class ChannelArguments;
 
 std::shared_ptr<Channel> CreateInsecureChannelFromFd(const std::string& target,
                                                      int fd) {
-  grpc::internal::GrpcLibrary init_lib;
-  init_lib.init();
   grpc_channel_credentials* creds = grpc_insecure_credentials_create();
   auto channel = CreateChannelInternal(
       "", grpc_channel_create_from_fd(target.c_str(), fd, creds, nullptr),
```

Comment:
We can add an `init_lib.shutdown();` before the return, and add an comment that that should be safe since `Channel` derives from `GrpcLibraryCodegen` which causes a `grpc_init()` in its constructor.

Reply:
That could work. Is the concern that `InsecureCredentials` construction may require gRPC init?

Reply:
Yes

Reply:
Alright, done.